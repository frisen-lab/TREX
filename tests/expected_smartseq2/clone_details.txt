clone_nr	clone_id	n_cells	clone_ids_per_cell
1	--------ATTTATAGGCTCAACATTCAAC	1	1.00
2	-----GCATGGGGGAGGCAATTGGAATTGG	1	2.00
3	---AGGTCGGAGTGTTGTCAGCCAAGAAGA	1	1.00
4	--GTACGTCTGTGGTTGGATTATATTGTAC	1	1.00
5	000000CGGACAACCGGGTTAGGGGCGTTA	1	1.00
6	00GTCGGGAAGAACCGAGTGGTGATAGTAC	1	3.00
7	00TCGACGTTATACGGCGGATGGCGTGTAC	1	1.00
8	AAACTTTGGGTTCATGACAATCCTCCACAG	1	1.00
9	AACCTTGCTATGTGTATAGGCGTTCTGTAC	1	1.00
10	AACGTCAATACGCTATCCCAAATTGTTAAC	1	1.00
11	ACCAATAGGAGCCTTAACCGTCGTGGGCCG	1	1.00
12	ACCTGCCCACGCACGGAAAGAGTGTGGGGC	1	2.00
13	AGACGGCACATGTGATTTGAATGGTTGTAC	1	1.00
14	AGCATTAAAGGACGAGGAATTCAGGCCTCC	2	1.00
15	AGGTCGGTCACTACCCGGAAGTGCATATTA	1	1.00
16	AGTCATTCTTACCTCAAGCGGATCTCCGGT	1	1.00
17	AGTGGTAAATGTGACTCTGGGCGCGGGGGT	1	1.00
18	AGTTGTTGCTGAGGAGTCGTTTATCTGGGG	1	2.00
19	ATAGCGCGGTCGGTGGACGATTTCGGATAC	1	1.00
20	ATTAGAATCAGTATGATTATTCATCTATAA	1	1.00
21	ATTCGTGTGCGCAGAATTGAGGGGAAGAG-	1	1.00
22	CAAAATACCGTACCTGCCCTGGGGGCCGAC	1	1.00
23	CCATGGGCAGCAGCAGGGTCTGCGCGGAAG	1	2.00
24	CCGAGTCTGCGGGGTGTCAACAGCTCAAGT	1	1.00
25	CCTGTAATCGGGGGACTTTTAAAGTGGGCG	1	1.00
26	CGCAGTGTAGTAACGTATCGTGGCTTGGTG	1	2.00
27	CGTAATCCTATCGCAATGAGGGTGCTGATC	1	2.00
28	CGTGGGCTCGGAGATGTGTATAAGAGACAG	1	2.00
29	CTACAATGCAGGACACAGACAGATTGCTGT	1	1.00
30	CTAGTAAAAGGAGTATATTCGG--------	1	1.00
31	CTCTTGATAGACCCTGTGGTAATATGTCCG	1	1.00
32	CTGCCACGAATAGCGAGTTTTGGGAGTTAC	1	6.00
33	GAAGGGGGCACGCACTAAATTCCGCAGGAC	1	3.00
34	GAAGTCTTACATTGGGGTAGAC--------	1	3.00
35	GACGTGGGGAGGTGCATTTGTGTATTATGA	1	1.00
36	GAGAATTTGGGTAGGACGCGCCCGAAACTT	2	1.00
37	GAGCGAAAATTGAAATATTGCTTCTTCGGT	1	4.00
38	GAGTGCAAGCCCAATCTTGTGTTTTAGTAC	1	2.00
39	GATGACACGGGGTTGACTAATGATGGTACC	1	1.00
40	GCCTATTCGTATTTAATACGGGCGAATTGG	1	1.00
41	GCGGACTGGATAAGCTGTGCGCAGAACTCG	1	3.00
42	GCGTGTGTAACAAGTGGCTCTACTGCAAGG	1	1.00
43	GCTTGGAACTTTTTGTCCCAGGATGAAACG	1	1.00
44	GGAAGGGGGTTGTAGGGATCGGTAATTCAA	1	1.00
45	GGATGACGGCGTTGGTCCCGG---------	1	2.00
46	GGCACCGGGAGGTATACGCAGGGTCAGGGC	1	1.00
47	GGCCGGCATCTCGTCATCTACTTTTGGCGA	1	4.00
48	GGGGTTGCATATATAGTTACGTGTCTGGTG	1	2.00
49	GTAAAATGAAGTGTGACATGTGCGGCTAGC	1	5.00
50	GTCTAGGCAGACACCATGGGTGCTGGGCAT	1	1.00
51	GTCTCAGTTTATAGTCCTTTGAAGTCGTAC	1	1.00
52	GTGGGAGTTAATGCATTAAAGCCCGCGTAC	1	1.00
53	GTTCTAGGTGCGACAGGAGGGTATCTGGTT	1	1.00
54	GTTGCTAACGCGAATAAACATACGTTAGAG	1	1.00
55	GTTGGTGCTGTAGTAATCCCGAGGGCTTAA	1	1.00
56	TAAAGGTTGGGGACGCAAACATTTTTGGGA	1	1.00
57	TACTAATAATGACATCCGAATGCGTTTTCG	1	1.00
58	TAGGTGTTTAGTGGTGTAAAGGGAACA---	1	1.00
59	TAGTGGGTGAGGGACGAGTCACGGTACCTT	1	1.00
60	TATATGTGCTCCGGCTCTCACTGGGGGTAC	1	1.00
61	TATTGGAGGTTGCTCACCATCTCGCAGCGC	1	2.00
62	TCAGAAGTTACACCGGGTGTTAGTG-----	1	2.00
63	TCAGAGCAGTATTGGTTAAGCGTGTAGGAG	1	1.00
64	TCAGGCTATACGTATAGGGCGAGTGCGTAG	1	1.00
65	TCAGTTCTGAGTGTAATTTCGTTATTTAGC	1	1.00
66	TCATAATGGGGGGCGAGGTATGAC------	1	1.00
67	TCGAAAGTAGAGGTGACCAAGTGAGATGAT	1	1.00
68	TCGATGGTACACTTTACAAGAATTAGGTGC	1	1.00
69	TCGATTATAATTCATGGGAGTGGTTAGTGT	1	2.00
70	TCGCCGGGGTCGTTTACGCCAGGTTTGGGC	1	1.00
71	TCGCGAGGTATATAGCGCAGTTGTAGAT--	1	2.00
72	TCGCGCTTGTATGTTTGGTATTCATATGAC	1	1.00
73	TCGCGTCGAACGTAGCACCTACCAGTGTGC	1	2.00
74	TCGCTACCTGTCTATACGCAGCGCGGGCGG	1	1.00
75	TCGGTCAGAAACCATGGGCTAGGGTTGGAG	1	1.00
76	TCGTCTGGCCCACCGAAGTCAGAGAACGTG	1	1.00
77	TCGTGTAGAGTGTGGTCTATGGGGATCATG	1	1.00
78	TCTAACGGCTCGCCCGGTGACCGTGCATAG	1	1.00
79	TCTCCAGGTTCCGGCGGGGAGAGGTTAGGG	1	1.00
80	TGAAATGGATGCGCAGCCGAAAAGCCGTAC	1	1.00
81	TGATACTGGATATGTTGTGCCAGAGGGCAC	1	1.00
82	TGATCAGGGTATAGGTTACATCAGAGGTAC	1	4.00
83	TGCGATACCGGAGTTTAGCGG---------	1	3.00
84	TGGACATAGCAGACTCCTACGTGGTAAAGG	1	1.00
85	TGGAGAAAGGGTCTTTTCACAATGTGATTG	1	3.00
86	TGTGGACGTACTATTTGGGCTCCGTTACCC	1	4.00
87	TGTGGATTGCACAGACCAGAGC--------	1	1.00
88	TTACCGGATATGTGTGGGTGCTTACTAGCG	1	5.00
89	TTAGCTAAGGTCTCTTAGGTTGGTATCGTG	1	1.00
90	TTGGGGCGTTAGAATGAATACCACTTGGCT	1	2.00
91	TTTCTAGGGGCTTCTTTTATTATGACAGGA	1	1.00
