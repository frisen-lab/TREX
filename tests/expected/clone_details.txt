clone_nr	clone_id	n_cells	clone_ids_per_cell
1	ACTAGGAGATTGACGGATCACCTTTGGTCG	2	1.00
2	GGCGGAGACGGCACACACCGAGGGTCAGTT	1	1.00
3	GGTCTCCCTATACCAACAGTATCGTCTCAA	8	1.00
4	TGTCAATCGTTCGGTTGAGCAAGATCTTAG	686	1.00
